addons/web/controllers/main.py

```diff
@@ -1998,7 +1998,7 @@ def report_download(self, data, context=None):
                     response = self.report_routes(reportname, docids=docids, converter=converter, context=context)
                 else:
                     # Particular report:
-                    data = dict(url_decode(url.split('?')[1]).items())  # decoding the args represented in JSON
+                    data = dict(url_decode(url.split('?')[1]).items()) if len(url.split('?')) > 1 else {}  # decoding the args represented in JSON
```

Comment:
Use `url_parse` instead, this way you can just use `url_parse(url).decode_query()`.

And the result can just be used as-is: [`url_decode`](https://werkzeug.palletsprojects.com/en/1.0.x/urls/#werkzeug.urls.url_decode) returns [a `MultiDict`](https://werkzeug.palletsprojects.com/en/1.0.x/datastructures/#werkzeug.datastructures.MultiDict) which can be used as-is, or alternatively we can pass `cls=dict` to ensure we get a `dict` out (e.g. avoid possible oddities with respect to `context`).